samples/bot-conversation/python/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "debug.onTaskErrors": "abort",
+    "json.schemas": [
```

Comment:
may remove `json.schemas` as there's no AAD app manifest in this project

Reply:
done